### DISCUSSION TITLE
Problem running template

### DISCUSSION THREAD

**Discussion by amartinAIDIMME** [coreui/coreui-free-react-admin-template#424] [General]
I downloaded the last release, and I run npm install and npm start, with no error at the server console.
But, the page does not load in the browser, and I have an error at the browser console:
> Uncaught SyntaxError: Unexpected token '<' (at index.js:10:3)
The error is in the < character of this line at index.js:
  <Provider store={store}>

This is the content of index.js (at the browser):
import React from 'react'
import { createRoot } from 'react-dom/client'
import { Provider } from 'react-redux'
import 'core-js'

import App from './App'
import store from './store'

createRoot(document.getElementById('root')).render(
  <Provider store={store}>
    <App />
  </Provider>,
)

**Comment by dinhthi12** [comment]
You can try clear node modules and npm i. Good luck
